### ISSUE TITLE
Running without root on macOS

### ISSUE THREAD

**Issue by sandstrom** [netbirdio/netbird#1551]
The current install flow requires sudo:

```sh
sudo netbird service install
sudo netbird service
```

Ideally we should be able to run netbird without root.

Some inspiration and potentially relevant links:

- https://news.ycombinator.com/item?id=33700136
- https://github.com/sandialabs/wiretap

Also, Docker Desktop has some examples on how to tackle the problem of running in user land (https://docs.docker.com/desktop/mac/permission-requirements/). The have a thin privileged agent, but the rest is user-space.

**Comment by myevit** [comment]
I don't think you can install/start service as non root user. You can probably run netbird as foreground service.

**Comment by jakob1379** [comment]
Systemctl runs fine as user services, that just needs to be recognized by the Netbird client and properly set the services in userspace instead

**Comment by lixmal** [comment]
You can run netbird without root: https://docs.netbird.io/how-to/netbird-on-faas

or the container image `netbirdio/netbird:rootless-latest`
